Source/PermissionTypes/SpeechRecognizer.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SpeechRecognizer.swift
+//  Permission
+//
+//  Created by BinaryBoy on 8/13/16.
+//  Copyright © 2016 delba. All rights reserved.
+//
+
+
+
+import Speech
+
+internal extension Permission {
+    var statusSpeechRecognizer: PermissionStatus {
+
+        let status = SFSpeechRecognizer.authorizationStatus()
+        
+        switch status {
+        case .Authorized:          return .Authorized
+        case .Restricted, .Denied: return .Denied
+        case .NotDetermined:       return .NotDetermined
+        }
+    }
+    
+    func requestSpeechRecognizer(callback: Callback) {
+
+        SFSpeechRecognizer.requestAuthorization { _ in
+                    callback(self.statusSpeechRecognizer)
+
+            }
+    }
+}
```

Comment:
Could you please adopt the project's coding style ? MIT License and number of spaces/indentation etc. ?
It seems like details but it greatly helps people who aren't familiar with the project to navigate quickly.